src/main/java/com/duke/bot/View.java

```diff
@@ -0,0 +1,11 @@
+package com.duke.bot;
+
+import java.util.List;
+
+public class View {
+    public void print(List<String> lines) {
```

Comment:
Would it be more useful to call this method printLines as opposed to print (since print only outputs the input Strings like in System.out.print?)?

Reply:
Makes sense, thanks for the input!